components/datetime/src/options/mod.rs

```diff
@@ -22,15 +25,19 @@
 //! );
 //! ```
 //!
-//! At the moment only the `Style` bag works, and we plan to extend that to support
-//! `ECMA 402` like components bag later.
+//! | Bag Type                                   | Support     |
+//! | ------------------------------------------ | ------------|
+//! | [`Style`](crate::options::style)           |implemented  |
+//! | [`Components`](crate::options::components) |unimplemented|
+//! | `Skeleton`                                 |unimplemented|
```

Comment:
I don't think we should separate Components and Skeleton since they are different representations of the same thing.